parm/config/config.anal

```diff
@@ -57,21 +57,22 @@ export OBERROR=$FIXgsi/prepobs_errtable.global
 
 
 # Use experimental dumps in EMC GFS v16 parallels
+export COMPONENT=${COMPONENT:-atmos}
```

Comment:
This is atmospheric analysis job.  Is it ever going to get a value of `COMPONENT` from anywhere else and will that be anything other than `atmos`?
```suggestion
export COMPONENT=atmos
```

Reply:
Probably won't be anything other than `atmos`. Will accept your suggested change, thanks!